### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
From #92 

I'm sorry. This is @trimental vs. https://github.com/crate-ci/typos 🤖 

_this is a devops person, I can find more problems in the code if you wish_

### CODE REVIEW

File: src/interpreter.rs [hunk 1]
```diff
@@ -181,7 +181,7 @@ impl HtmlInterpreter {
         let span_color = native_color(self.theme.code_color, &self.surface_format);
         let mut tok = Tokenizer::new(self, TokenizerOpts::default());
 
-        for md_string in reciever {
+        for md_string in receiver {
```
Comment (szepeviktor) [inline src/interpreter.rs:184]: Change in code!

File: src/positioner.rs [hunk 2]
```diff
@@ -249,12 +249,12 @@ impl Positioner {
 #[derive(Debug)]
 pub struct Spacer {
     pub space: f32,
-    pub visibile: bool,
+    pub visible: bool,
```
Comment (szepeviktor) [inline src/positioner.rs:252]: Change in code!

**Comment by szepeviktor** [comment]
➕ 1, there is _syntect highlighting_ in README

**Review by CosmicHorrorDev — APPROVED** [review]
To be fair, some of those typos are mine :sweat_smile: 

Looks great. Thanks for the changes!

**Pull request merged**